src/main/java/XbeLoader/XbeLoader.java

```diff
@@ -554,6 +554,10 @@ private void processImports(Program program, TaskMonitor monitor, MessageLog log
 		SymbolTable symbolTable = program.getSymbolTable();
 
 		Address address = kernelThunkTableAddr;
+		if (address.getOffset() == 0) {
+			// No kernel thunk table, skip processing step
```

Comment:
If I remember correctly, this situation will crash some of the MS tools like VerifierX (which runs as default for many people on XDKs).

So it would make sense to print a warning here.